cassandra_migrate/migrator.py

```diff
@@ -579,6 +579,7 @@ def reset(self, opts):
         self.cluster.refresh_schema_metadata()
 
         opts.force = False
+        opts.force_in_progress = False
```

Comment:
why this?

Reply:
because reset is about to call migrate, and force_in_progress is now an argument migrate takes but which reset doesn't take

i didn't test reset on my previous pr and it turns out it needs that to explicitly be false to not choke